hicommonwealth/commonwealth
# Fix UI issues with community members page

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/8477

## Description of Changes
The original ticket for this pr was about fixing

> 1. Inifite look when searching numbers like 0x1
> 2. Fix page result ordering issue in table view 
> 3. Fix not in group filters 
> 4. When searching by address (like 3B9 dillon's) the server returns 27 results 

but we paused the PR per profile migration work. 

~~After resuming work, I wasn't able to reproduce 2, 3 and 4. 1 was already fixed in #8463.~~ Thanks to Roger, I was able to finally repro `2` and `3`.

- This pr removes unnecessary mapping/filtering of members from the frontend community members page
- Fixes `2` and `3`
- Updates AllowList groups filter types and some more types

## "How We Fixed It"
N/A

## Test Plan
- Visit community members page
- Open network console and make a member search
- Inspect API results from the members API and verify you see the correct count of members returned, rendered in the members table.
- Verify that sorting the table, correctly sorts the data, and makes new API calls per filter.
- Verify that the groups filter correctly filters out members per the filter criteria
- Test Allowlist - visit create group page with an admin account and verify you are correctly able to filter members in table using the groups filter

## Deployment Plan
N/A

## Other Considerations
N/A
